instrumentation/grape/README.md

```diff
@@ -30,6 +41,32 @@ OpenTelemetry::SDK.configure do |c|
 end
 ```
 
+### Configuration options
+
+#### `:ignored_events` (array)
+
+Indicate if any events should not produce spans.
+
+- Accepted values: `:endpoint_render`, `:endpoint_run_filters`.
```

Comment:
why isn't `format_response` an option ?

TBH `endpoint_run` could also be excluded and then the generated spans are not parented at all (except the root span parent).

Reply:
Because I forgot to add it 😅 

> TBH endpoint_run could also be excluded and then the generated spans are not parented at all (except the root span parent).

Yes, you're right, but I didn't add it as an option because since it's the "main" span, I thought it wouldn't make a lot of sense to ignore this one, which is also the one who has all the attributes, and be left with all the orphans 😅